src/hotspot/share/opto/vectornode.hpp

```diff
@@ -890,6 +890,7 @@ class LoadVectorNode : public LoadNode {
 // Load Vector from memory via index map
 class LoadVectorGatherNode : public LoadVectorNode {
  public:
+  enum { Offsets = 3 };
```

Comment:
Just wondering about the naming convention here because the corresponding constructor argument is called `indices`. Should that be consistent?

Reply:
Also, where is this used?

Reply:
You're right, they are always called "indices" even in the Vector API documentation. I've renamed them: offsets -> indices.
I've also removed all unused enums (in `LoadVector...Node` classes).